discoverd/server/handler.go

```diff
@@ -0,0 +1,399 @@
+package server
+
+import (
+	"encoding/json"
+	"math/rand"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/julienschmidt/httprouter"
+	"github.com/flynn/flynn/discoverd/client"
+	hh "github.com/flynn/flynn/pkg/httphelper"
+	"github.com/flynn/flynn/pkg/sse"
+	"github.com/flynn/flynn/pkg/status"
+	"github.com/flynn/flynn/pkg/stream"
+)
+
+// StreamBufferSize is the size of the channel buffer used for event subscription.
+const StreamBufferSize = 64 // TODO: Figure out how big this buffer should be.
+
+// NewHandler returns a new instance of Handler.
+func NewHandler() *Handler {
+	h := &Handler{router: httprouter.New()}
+
+	h.router.Handler("GET", status.Path, status.SimpleHandler(func() error { return nil }))
```

Comment:
The handler can be replaced with `status.HealthyHandler`, though add a comment to check the status of the consensus cluster.